modules/worker_runner/templates/worker-runner.sh.erb

```diff
@@ -20,6 +20,7 @@ rm <%= @data_dir %>/tasks-resolved-count.txt
 
 # Clear up previous task dirs
 rm -rf /opt/worker/tasks/task_*
+find /private/var/folders/ -path '*/C/org.mozilla.*' -type d -o -path '*/C/org.mozilla.*/*' -type d -exec rm -rf {} +
```

Comment:
Should we as well clean up the `Download` folder? Maybe a new section for both makes sense given that both are not related to the `task dirs`.

Reply:
That's in this PR handled by a different tool https://github.com/mozilla-platform-ops/ronin_puppet/pull/774